qiime2/sdk/plugin_manager.py

```diff
@@ -93,18 +93,38 @@ def _integrate_plugin(self, plugin):
     # databases rollback/commit
 
     @property
+    def importable_formats(self):
+        """Return set of formats that are importable.
```

Comment:
(Following conversation I am changing it a dict for user convenience and API transparency)